main.go

```diff
@@ -202,8 +192,23 @@ func wrappedMain() int {
 			fmt.Fprintf(os.Stderr, "Packer failed to initialize UI: %s\n", err)
 			return 1
 		}
+	} else {
+		var TTY packer.TTY
+		if !inPlugin {
+			var err error
+			TTY, err = tty.Open()
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "No tty available: %s\n", err)
+			}
+		}
+		// Setup the UI if we're being machine-readable
```

Comment:
this comment isn't making sense to me -- isn't this whole else statement when we are _not_ being machine readable?

Reply:
Oh, right I just moved that block without paying too much attention to the comment and it came along. Ok, I'll put it up there.